libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -30,7 +30,6 @@ extern const AP_HAL::HAL& hal;
 #include <AP_Math/AP_Math.h>
 
 #include "RC_Channel.h"
-#include <GCS_MAVLink/GCS.h>
```

Comment:
Still need this include; `GCS_SEND_TEXT` is used

Reply:
ok, keep it there.